src/renderer/store/handlers/directMessages.ts

```diff
@@ -41,7 +41,7 @@ export type DirectMessagesStore = DirectMessages
 
 export const initialState: DirectMessagesStore = {
   users: {
-    '02dc8264c555d46b3f6b16f1e751e979ebc69e6df6a02e7d4074a5df981e507da2': {
+    holmes: {
```

Comment:
We should remove any stuff about hardcoding my username. It was a hack for the previous first time user experience and we don't need it anymore!

Reply:
I've missed that part, I'll remove that.